src/common/MessageProcessor.sol

```diff
@@ -208,15 +198,13 @@ contract MessageProcessor is Auth, IMessageProcessor {
                 m.isIncrease
             );
         } else if (kind == MessageType.UpdateShares) {
+            // TODO: Remove price from hub
+            D18 price = d18(1, 1);
```

Comment:
@reviewer 

will be removed in https://github.com/centrifuge/protocol-v3/pull/242